core/templates/pages/skill-editor-page/services/skill-editor-state.service.ts

```diff
@@ -171,8 +180,12 @@ export class SkillEditorStateService {
     return this._skillIsBeingLoaded;
   }
 
-  getAssignedSkillTopicData(): string {
-    return this.assignedSkillTopicData;
+  /**
+   * _assignedSkillTopicData will be null (the initial value)
+   * if loadSkill() fails to initialize it due to any reasons.
+   */
+  getAssignedSkillTopicData(): { [topicName: string]: string } | null {
```

Comment:
Use the `AssignedSkillTopicData` type here.

Reply:
Done!